src/server/auth/cmds/cmds_test.go

```diff
@@ -275,15 +317,6 @@ func TestMain(m *testing.M) {
 			panic(err.Error())
 		}
 	}
-	time.Sleep(time.Second)
```

Comment:
Deactivating auth is synchronous now, this was adding latency to every test with no benefit.

Reply:
Good catch!